_posts/en/newsletters/2020-04-15-newsletter.md

```diff
@@ -0,0 +1,92 @@
+
+## News
+
+- **Proposal for using one BIP32 keychain to seed multiple child keychains:**
+  several weeks ago, Ethan Kosakovsky [posted][kosakovsky post] to the
+  Bitcoin-Dev mailing list a [proposal][kosakovsky proposal] for using
+  one [BIP32][] Hierarchical Deterministic (HD) keychain to create seeds
+  for child HD keychains that can be used in different contexts.  This
+  may seem unnecessary given that BIP32 already provides extended
+  private keys (xprvs) that can be shared between signing wallets.  The
+  problem is that many wallets don't implement the ability to import
+  xprvs---they only allow importing either an HD seed or some precursor
+  data that is transformed into the seed (e.g.  [BIP39][] or [SLIP39][]
+  seed words).
+
+    Kosakovsky's proposal is to create a super-keychain whose child keys
+    are transformed into seeds, seed words, or other data that can be
+    input into various wallets' HD keychain recovery fields.  That way a
+    user with multiple wallets can backup all of them using the just the
+    super-keychain's seed (plus the derivation paths and the library for
+    transforming determinsitic entropy into input data).
+
+    {% assign img="/img/posts/2020-04-subkeychains.dot.png" %}
+    [![Using one BIP32 keychain to seed child BIP32 keychains]({{img}})]({{img}})
+
+    Reaction to the proposal was mixed, but this week one hardware wallet
+    manufacturer [stated][novak post] their intent to implement support
+    for the protocol and requested additional review of the proposal.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 0.20.0rc1][bitcoin core 0.20.0] the first release
+  candidate for the next major version of Bitcoin Core is available for
+  testing by experienced users.  Please [report][bitcoin core issue] any
+  problems you discover so that they can be fixed before the final
+  release.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [LND #4075][] allows invoices to be created for payments greater than the
+  previous limit of around 0.043 BTC. The network-wide HTLC limit of 0.043 BTC
+  prevents payments greater than that amount over a single channel, but with the
+  implementation of sending [multipath payments][topic multipath payments] also
+  merged in [LND this week](#lnd-3967), invoices can be issued for an aggregate
+  payment amount greater than 0.043 BTC, which the sender then splits into
+  partial payments, each of which is smaller than 0.043 BTC.
+
+- [LND #3967][] adds long-awaited support for sending [multipart payments][topic
```

Comment:
I think we should standardize to either 'multipath' or 'multipart' in our newsletters. I've changed this to 'multipath', since that's the canonical name for our topic page, but happy to debate which term we should standardize on.

Reply:
> I think we should standardize to either 'multipath' or 'multipart' in our newsletters.

We're de facto standardized on "multipath".  See https://github.com/bitcoinops/bitcoinops.github.io/blame/master/_topics/en/multipath-payments.md#L6-L8 and the absence of any normal newsletter text calling them multipart

```text
$ git grep -i 'multi-*part\>'
_topics/en/multipath-payments.md:## calls them multi-part payments, no C-Lightning sources yet but Rusty and
_topics/en/multipath-payments.md:  - Multipart payments
_topics/en/transaction-pinning.md:  management more difficult in multipart contract protocols (such as LN).
```